docs/src/booklet.md

```diff
@@ -32,16 +36,23 @@ ToQUBO.PBO.derivative
 ToQUBO.PBO.gradient
 ToQUBO.PBO.gap
 ToQUBO.PBO.sharpness
-ToQUBO.PBO.discretize
 ToQUBO.PBO.relaxed_gcd
 ```
 
+### A Primer on Submodularity
+A set function ``f : 2^{S} \to \mathbb{R}`` is said to be submodular if
+
+```math
+f(X \cup Y) + f(X \cap Y) \le f(X) + f(Y) \forall X, Y \subset S
+```
+
+holds.
+
```

Comment:
Why do you add this primer? Do you use it later?

Reply:
This primer is important to set the basis for deciding which quad to use. It just happened that we didn't give that much attention to implementing multiple quads and this decision became trivial between the methods we have right now.

Reply:
Added to #63